### ISSUE TITLE
ensure kea configuration include authoritative true

### ISSUE THREAD

**Issue by sarcasticadmin** [socallinuxexpo/scale-network#805]
## Description

Follow up from: https://github.com/socallinuxexpo/scale-network/issues/674#issuecomment-2520050029

Originally from @owendelong :
```
We should make sure that we set authoritative true; in the KEA configuration files.

KEAs default behavior is to grant the requested lease unless it's been reissued. Most of the tweaks are to change lease expiration behavior, which we have pretty well tuned for the environment.
```

## Acceptance Criteria

- Ensure that `kea` config for both dhcp severs is `authoritative true`

**Comment by owendelong** [comment]
I don't think this is related to lease affinity. The use of the authoritative flag in the configuration for the DHCP4 server relates to DHCP4 reservations more than to leases. Mostly this isn't a problem in our environment because we don't make reservations that overlap our pools (yes, there are environments where this is common practice and it's basically encouraged in many AD administration manuals, so thanks Micr0$0ft for inflicting yet another bad habit on the world).

However, one area where this could come into play for us is if a host has an address, and we change the reservation, then reboot the host, it may ask for its old address, and if the authoritative flag isn't set, kea may grant the old address unless it's already been recycled, rather than forcing the new reservation. With authoritative set, kea should force the reservation.

For DHCPv6, ironically, the authoritative flag is unnecessary as that is the required behavior in DHCPv6.
